pkg/transaction/transaction.go

```diff
@@ -152,6 +155,19 @@ func (t *transactionService) waitForAllPendingTx() error {
 
 	return nil
 }
+func (t *transactionService) setAccumulativeFee(gasTipCap, gasFeeCap *big.Int, gasLimit uint64) {
```

Comment:
this is not an accurate way to compute the fee.
What needs to be done is check the transaction info after the transaction is mined in a block, and get the price and amount of gas used for the transaction.

Reply:
I believe the receipt has some info, if not, we need to figure out another way

Reply:
```
type Receipt struct {
	// Consensus fields: These fields are defined by the Yellow Paper
	Type              uint8  `json:"type,omitempty"`
	PostState         []byte `json:"root"`
	Status            uint64 `json:"status"`
	CumulativeGasUsed uint64 `json:"cumulativeGasUsed" gencodec:"required"`
	Bloom             Bloom  `json:"logsBloom"         gencodec:"required"`
	Logs              []*Log `json:"logs"              gencodec:"required"`
	TxHash          common.Hash    `json:"transactionHash" gencodec:"required"`
	ContractAddress common.Address `json:"contractAddress"`
	GasUsed         uint64         `json:"gasUsed" gencodec:"required"`
	BlockHash        common.Hash `json:"blockHash,omitempty"`
	BlockNumber      *big.Int    `json:"blockNumber,omitempty"`
	TransactionIndex uint        `json:"transactionIndex"`
}
```
The receipt doesn't have any info.

Reply:
I was not able to find fee data anywhere else. Maybe @swarmHaseeb can explain.